README.md

```diff
@@ -24,6 +24,8 @@ provider:
               - 222222222222 # principal as accountId
               - 'arn:aws:iam::333333333333:root' # principal as ARN
               - Fn::Import: cloudformation-output-arn-2 # principal as CloudFormation Output Value ARN
+            allowTagSession: True # can optionally be defined to include sts:TagSession in assume role policy
```

Comment:
Is there any harm in just always adding it?

Reply:
I was avoiding breaking changes.  I personally don't care.  :)

Reply:
It's not breaking as it's adding an action, not removing it. Plus, that action should probably always be there.

Reply:
I would generally agree, but from a least privilege standpoint it's a gray area.  I'm good with whatever you gentlemen decide.  I'm just interested in switching the lms to 12 hours and tagging.

Reply:
I'm OK with leaving it as is